react/javascript/demo-extension-sdk/src/demos/Clipboard.jsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 Looker Data Sciences, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+import React, { useContext, useState } from 'react'
+import { Button, MessageBar, SpaceVertical, Tooltip } from '@looker/components'
+import { ExtensionContext2 } from '@looker/extension-sdk-react'
+import { DemoWrapper } from '../components'
+
+const title = 'Clipboard write'
+const description = `Demonstrates how to write to the system clipboard.
+
+The Looker Extension SDK API allows an extension to write to the system
+clipboard.
+
+The \`use_clipboard\` entitlement must be set to write to the clipboard.`
```

Comment:
This was introduced in 21.8, right? I think it's always worth mentioning that either in the README or the comment here or something, so that we don't invite "why isn't this working!" for folks on earlier versions.

But just a preference.

Reply:
Good catch. Added a new hook to verify the looker version and use hook result display a message that it's not supported. Also added version required to component documentation.